manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -4188,136 +4188,197 @@
   </sect1>
 
   <sect1 id="trans-sched-slr">
-    <title>Since Last Run Assistant</title>
-
-    <para>The <guilabel>Since Last Run</guilabel> assistant is run automatically when &app; is started. It is
-      used to enter into the register any transactions that are due to be automatically entered (see
-      <xref linkend="trans-sched"></xref>). The run on &app; start can be altered in the
-      <link linkend="prefs-sched">Scheduled Transactions</link> tab of the <guilabel>&appname;
-      Preferences</guilabel>. To run the <guilabel>Since Last Run</guilabel> assistant manually, go
-      to
+    <title>Der <quote>Seit letztem Aufruf</quote> Dialog</title>
+
+    <para>Der <guilabel>Seit letztem Aufruf</guilabel> Dialog wird automatisch ausgeführt, während &app;
+      startet. Er dient der Eintragung von Buchungen in das Kontobuch, die automatisch erfaßt
+      werden sollen (siehe <xref linkend="trans-sched" />). Die Ausführung während des Starts kann
+      in den <guilabel>&appname; Eigenschaften</guilabel> auf dem Reiter <xref linkend="prefs-sched" />
+      angepasst werden. Möchten Sie den Assistenten manuell auszuführen, gehen Sie im Menu auf
       <menuchoice>
-        <guilabel>Actions</guilabel><guisubmenu>Scheduled Transactions</guisubmenu>
-        <guimenuitem>Since Last Run...</guimenuitem>
+        <guimenu>A<accel>k</accel>tionen</guimenu><guisubmenu>Term<accel>i</accel>nierte
+        Buchungen</guisubmenu> <guimenuitem>Seit <accel>l</accel>etztem Aufruf…</guimenuitem>
       </menuchoice>
+      .
     </para>
 
-    <para>In the <guilabel>Since Last Run</guilabel> assistant window you can see three columns:
-    </para>
-
-    <variablelist termlength="8">
-      <varlistentry>
-        <term><guilabel>Transaction</guilabel></term>
-
-        <listitem>
-          <para>The name assigned in the <guilabel>Scheduled Transaction Editor</guilabel> that identifies the
-            scheduled transaction.
+    <figure pgwide="1">
+      <title>Der <quote>Seit letztem Aufruf</quote> Dialog</title><screenshot>
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="figures/Tools_SinceLastRun.png" srccredit="Christian Wehling" />
```

Comment:
Komischerweise sehe ich das Bild in epub, mobi, aber nicht in yelp oder mit dem HTML-Browser.